recipes/tiddit/meta.yaml

```diff
@@ -1,45 +1,45 @@
-{% set version = "2.12.2" %}
+{% set version = "3.0.0" %}
 
 package:
   name: tiddit
   version: '{{ version }}'
 
 source:
-  url: https://github.com/SciLifeLab/TIDDIT/archive/TIDDIT-{{ version }}.tar.gz
-  sha256: 679d06769077afb0431dfb780f2a7d3311c0d9c389301dcf55ab46ee360d91df
-  patches:
-    - TIDDIT.py.patch
+  url: https://github.com/J35P312/TIDDIT/archive/TIDDIT-{{ version }}.tar.gz
+  sha256: 92b0ff9e8c85c172b2a0bae52d6623f3f16b926302ddd2c5b96a6119562f994b
 
 build:
-  number: 1
+  number: 0
+  entry_points:
+    - tiddit = tiddit.__main__:main
+  script: python -m pip install --no-deps --ignore-installed .
   skip: True  # [osx]
 
 requirements:
   build:
-    - make
+    - {{ compiler('cxx') }}
```

Comment:
Are you sure this is needed? Looking through the source files it looks like the C compiler may suffice.